chanbackup/pubsub.go

```diff
@@ -49,7 +50,7 @@ type ChannelEvent struct {
 type ChannelSubscription struct {
 	// ChanUpdates is a read-only channel that will be sent upon once the
 	// primary channel state is updated.
-	ChanUpdates <-chan ChannelEvent
+	ChanUpdates chan ChannelEvent
```

Comment:
why is it now send/receive? comment should be updated if it's no longer read-only

Reply:
Made it easier to write some unit tests